alphastats/gui/pages/05_LLM.py

```diff
@@ -90,23 +91,27 @@ def select_analysis():
     st.session_state["gpt_submitted_clicked"] = 0
     st.session_state["gpt_submitted_counter"] = 0
 
-c1, c2 = st.columns((1, 2))
 
+st.markdown("#### Configure LLM")
```

Comment:
Maybe make this a card, so it can be hidden away.

Reply:
then people will forget to do it .. (workflow will be changed anyway soon)